collections/truck-detection/README.MD

```diff
@@ -125,6 +125,17 @@ The **detection method** exploits small sensing offset of different wavelength t
    </tbody>
 </table>  
 
+### Paramters usage
+Area of Interest: The area where the algorithm requests the Sentinel-2 data and detects trucks. Noted that the algorithm requests ALL Sentienl-2 data available inside of area of interest no matter there are roads or not.<br />
+<br />
+Time range: The time range that the algorithm requests the Sentinel-2 data and detects trucks. Noted that the algorithm requests ALL available data/acquisitions within the time range.<br />
+<br />
+OSM Values: The road types that the algorithm detects trucks. Currently motorway, trunk, and primary are supported. The OSM Values determine the valid area and mask data outside of selected roads.<br />
+<br />
+Max. cloud coverage: The algorithm computes the cloud coverage of the pixels located over the network of roads (pixel intersects a road) and we compute the % of cloudiness by dividing the number of cloudy pixels by the total number of pixels intersecting roads. There is a trade-off between a high value that can generate a lot of false positives because of the same signature of cloud edges as trucks, and a low value that can loose a lot of traffic detected resulted from filtering too much.<br />
```

Comment:
`pixel intersects a road` 
pixels intersecting roads ...      (sounds better)

`and we compute the % of`
and computes the % of ...        (no need to use 'we')